octobot/constants.py

```diff
@@ -44,12 +45,21 @@
 DEFAULT_COMMUNITY_WS_URL = "ws://community2.octobot.cloud/"   #TODO
 OCTOBOT_COMMUNITY_URL = os.getenv("COMMUNITY_SERVER_URL", DEFAULT_COMMUNITY_URL)
 OCTOBOT_COMMUNITY_FEED_URL = os.getenv("OCTOBOT_COMMUNITY_WS_URL", f"{DEFAULT_COMMUNITY_WS_URL}cable")
-OCTOBOT_COMMUNITY_AUTH_URL = f"{OCTOBOT_COMMUNITY_URL}spree_oauth/token"
+OCTOBOT_COMMUNITY_FEED_URL = os.getenv("OCTOBOT_COMMUNITY_MQTT_URL", "iot.fr-par.scw.cloud")    #TODO
 OCTOBOT_COMMUNITY_ACCOUNT_URL = f"{OCTOBOT_COMMUNITY_URL}api/v2/storefront/account"
 OCTOBOT_COMMUNITY_PACKAGES_URL = f"{OCTOBOT_COMMUNITY_URL}api/v2/storefront/tentacle/packages"
 OCTOBOT_COMMUNITY_SUPPORTS_URL = f"{OCTOBOT_COMMUNITY_URL}api/v2/storefront/supports"
 OCTOBOT_COMMUNITY_FETCH_FEED_IDENTIFIER_URL = f"{OCTOBOT_COMMUNITY_URL}api/v2/storefront/feeds/id"
 COMMUNITY_FEED_CURRENT_MINIMUM_VERSION = "1.0.0"
+COMMUNITY_FEED_DEFAULT_TYPE = octobot.enums.CommunityFeedType.MQTTFeed
+
+COMMUNITY_BACKEND_API_URL = "https://octo-bot.swell.store/api"
+COMMUNITY_GQL_BACKEND_API_URL = "https://eu-west-1.aws.realm.mongodb.com/api/client/v2.0/app/community-backend-production-wgdgz/graphql"
+COMMUNITY_GQL_AUTH_URL = "https://realm.mongodb.com/api/client/v2.0/app/community-backend-production-wgdgz/auth/providers/api-key/login"
+COMMUNITY_GQL_REFRESH_AUTH_URL = "https://realm.mongodb.com/api/client/v2.0/auth/session"
+COMMUNITY_BACKEND_AUTH_URL = f"{COMMUNITY_BACKEND_API_URL}/account/login"
+COMMUNITY_TOKEN = os.getenv("COMMUNITY_TOKEN", "pk_mvKn5XznfVVSyp3XZlgodBKWlRhazeSu")
```

Comment:
Did you commit your own token or it's a public one?

Reply:
it's public :)